troposphere/cloudfront.py

```diff
@@ -401,6 +404,130 @@ class OriginRequestPolicy(AWSObject):
     }
 
 
+class AccessControlAllowHeaders(AWSProperty):
+    props = {
+        "Items": [(str, True)]
```

Comment:
The form `[(str, True)]` should be `([str], True)`. In other words, the tuple is (property_type, required_bool) and the type is an array of str `[str]`.